pkg/guestconversion/guestconversion.go

```diff
@@ -42,9 +43,22 @@ func MakeGuestConversionJobSpec(vmSpec *v1.VirtualMachine, libvirtConfigMap *cor
 							Image:           virtV2vImage,
 							VolumeMounts:    volumeMounts,
 							ImagePullPolicy: imagePullPolicy,
+							// Request access to /dev/kvm via Kubevirt's Device Manager
+							Resources: corev1.ResourceRequirements{
+								Requests: corev1.ResourceList{
```

Comment:
IIUC, the `Requests` is not necessary. In my tests, it `Limits` was enough.

Reply:
Indeed, according to https://kubevirt.io/api-reference/master/definitions.html#_k8s_io_api_core_v1_resourcerequirements it sets the Requests to be equal to Limits if Requests is not set. Thus, this is redundant and can be removed.